spec/classes/redis_spec.rb

```diff
@@ -81,6 +81,21 @@ class { 'redis::globals':
             end
           end
         end
+
+        describe 'with manage_dnf_module true', if: facts[:os]['family'] == 'RedHat' && facts[:os]['release']['major'].to_i > 7 do
```

Comment:
I'm never a fan of `> x`
```suggestion
        describe 'with manage_dnf_module true', if: facts[:os]['family'] == 'RedHat' && facts[:os]['release']['major'].to_i >= 8 do
```